stanford-earth/SE3_BLT
# EARTH-144

# READY FOR REVIEW 

# Summary
- Adds and enables the stanford_department content type

# Needed By (Date)
- End of this sprint (July 29th)

# Urgency
- Moderate

# Steps to Test

0. Save any work in progress you may have.
1. Check out this branch.
2. Run `composer update` to download the new module
3. Run `drush cim` to import default config
4. Run `drush csim local` to import local config
4b. Run `drush features-import-all` to import the feature config
5. Navigate to `/node/add` to confirm there is a Department option
6. Create a new department page by adding a few paragraphs. There should be all of the currently installed options available.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/EARTH-145 Department node template
- @koppieesq 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
